accepted/calc-functions.md

```diff
@@ -294,16 +294,25 @@ This algorithm takes a calculation `calc` and returns a number or a calculation.
   > otherwise calculations' arguments must match the expected number.
 
 * If `calc`'s name is `"sin"`, `"cos"`, `"tan"`, `"asin"`, `"acos"`, `"atan"`,
```

Comment:
I believe functions `sin`, `cos`, and `tan` should accept units (maybe only angle units)? E.g cos math function: 
```
final _cos = _function(
    "cos",
    r"$number",
    (arguments) => SassNumber(math.cos(arguments[0]
        .assertNumber("number")
        .coerceValueToUnit("rad", "number"))));
```

Reply:
Good point, I'll call those out in the note below.